### PR TITLE
Fix disable_lval_as_an_expression

### PR DESCRIPTION
**Pull request by lexidor**

### CODE REVIEW

File: src/Nuxed/Console/HelpScreen.hack [hunk 1]
```diff
@@ -78,7 +78,8 @@ final class HelpScreen {
 
     $retval[] = $this->renderUsage();
     if (!C\is_empty($this->arguments->all())) {
-      if ($output = $this->renderSection($this->arguments)) {
+      $output = $this->renderSection($this->arguments);
+      if ($output) {
```
Comment (lexidor) [inline src/Nuxed/Console/HelpScreen.hack]: P.S. string(1)"0" is falsey.

Comment (lexidor) [reply src/Nuxed/Console/HelpScreen.hack]: So `$output` can't be "" or "0", was this expected?

Comment (azjezz) [reply src/Nuxed/Console/HelpScreen.hack]: `renderSection` would never return "0", however, this should probably be `'' !== $output` ( not `!Str\is_empty($output)` )

File: src/Nuxed/Console/Event/CommandEvent.hack [hunk 2]
```diff
@@ -18,14 +18,16 @@ final class CommandEvent extends Event {
...
   }
 
   /**
    * Enable the command, so it would run.
    */
   public function enableCommand(): bool {
-    return $this->commandShouldRun = true;
+    $this->commandShouldRun = true;
+    return $this->commandShouldRun;
   }
```
Comment (azjezz) [inline src/Nuxed/Console/Event/CommandEvent.hack]: `disableCommand()` and `enableCommand()` should actually return `void`, i don't know why i did this in the first place :thinking: 

could you please change it ?

**Comment by azjezz** [comment]
`hhvm/hhast` is broken. let's wait for it to get fixed first.

File: src/Nuxed/Console/Event/CommandEvent.hack (hunk 2 above)
Comment (lexidor) [reply src/Nuxed/Console/Event/CommandEvent.hack]: It will be a `return;` in no time.

**Comment by azjezz** [comment]
`hhvm/hhast` 4.25.0 is broken, causing build for hhvm 4.25 to fail.

**Comment by azjezz** [comment]
Thank you @lexidor :heart:

**Pull request merged**